notation/stack.py

```diff
@@ -0,0 +1,24 @@
+"""Implementation of a stack of cards."""
+from typing import List
+
+import tomlkit
+
+from notation.card import Card
+
+
+# pylint: disable=too-few-public-methods
+class Stack:
+    """A list of cards."""
+
+    def __init__(self, cards: List[Card]):
+        """
+        Construct a stack of cards.
+
+        :param cards: A list of cards.
+        """
+        self._card_list: List[Card] = cards
+
+    def dumps(self) -> str:
+        """Return the stack as TOML string."""
+        card_dump = {'cards': [card.dict() for card in self._card_list]}
+        return tomlkit.dumps({'stack': card_dump})
```

Comment:
a stack class is not very pythonic, because this could just be a list.
the wrapping is nice for encapsulation though. so its fine i guess.

Reply:
but why is the representation `{stack={cards=[<card-list>]}}` ? why wrap it two times?
it could just be written as an array of cards

Reply:
Yeah, I think I had something else in mind originally where I needed an array. I try to change to an array.